### ISSUE TITLE
db configuration change - how to migrate data

### ISSUE THREAD

**Issue by aggarwalankush** [driftyco/ionic-storage#3]
Hi @mlynch 

I see you are using `_ionickv` table name at [Line40](https://github.com/driftyco/ionic-storage/blob/master/src/storage.ts#L40) which earlier(in beta 11, ionic-angular) was `kv`. How can I migrate my data from `kv` to `_ionickv` if I start using rc0? Also, if in future this db configuration changes, it will again loss data in old table. 

```
  this._db.config({
      name        : '_ionicstorage',
      storeName   : '_ionickv'
    });
```

**Comment by ghenry22** [comment]
actually in the earlier betas the name was configurable.  Can the db name still be configured and if so how?  Alternatively can you add back in the support for setting the DB name?

**Comment by mlynch** [comment]
Coming in the next release, we will make this configurable. Thanks for the input on this!

**Comment by mlynch** [comment]
Woops, forgot to close this. This was fixed and is available in 1.1.7: https://github.com/driftyco/ionic-storage#configuring-storage-new-in-117
